### PR TITLE
Better lights2

### PR DESCRIPTION
**Pull request by salvobonsma**
This Pull Request closes issue #

Description of the changes:
 - 

 - [ ] I have labelled this issue with the priority and correct subsystem.
 - [ ] I set the person that I worked on this with as a reviewer (who will need to approve the changes)

### CODE REVIEW

**Review by FlareTheSaberDragon — DISMISSED** [review]
Good job you sussy baka
download

**Comment by clavinohou** [comment]
good job guys

File: .idea/.gitignore
Comment (LinusSkucas) [inline .idea/.gitignore:1]: Can you delete all the .idea files?

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# ChargedUp
\ No newline at end of file
+# **ChargedUp**
```
Comment (LinusSkucas) [inline README.md:1]:
```suggestion
#  ChargedUp
```
It's unneeded

File: src/main/java/frc/robot/subsystems/LightsSubsystem.java [hunk 2]
```diff
@@ -26,14 +28,33 @@ public enum LightPattern {
 
   AddressableLED addressableLed = new AddressableLED(Constants.LED_STRIP_PORT);
   AddressableLEDBuffer ledBuffer = new AddressableLEDBuffer(Constants.LED_STRIP_BUFFER_SIZE);
+  SendableChooser<String> lightPatternChooser = new SendableChooser<>();
+  String lastSelectedLightPattern;
   int rainbowFirstLedHue = 0;
 
   public LightsSubsystem() {
     addressableLed.setLength(ledBuffer.getLength());
+
```
Comment (LinusSkucas) [inline src/main/java/frc/robot/subsystems/LightsSubsystem.java:37]: Has this been tested?

**Review by LinusSkucas — DISMISSED** [review]

**Review by Iron-Riders-Student — APPROVED** [review]

**Review by FlareTheSaberDragon — APPROVED** [review]
Good job you sussy baka
download

**Pull request merged**